eth/api_backend.go

```diff
@@ -324,7 +324,11 @@ func (b *EthAPIBackend) SendTx(ctx context.Context, signedTx *types.Transaction)
 	if err := ctx.Err(); err != nil {
 		return err
 	}
-	return b.eth.txPool.AddLocal(signedTx)
+	if err := b.eth.txPool.AddLocal(signedTx); err != nil {
```

Comment:
Was torn on whether this should be handled by the `txPool` or outside of it. I thought outside of it seemed cleaner/better separation of concerns.